index.html

```diff
@@ -0,0 +1,15 @@
+<html>
+ <body>
+  <script>
+   while (true) {
+    let name = prompt("日本の首都は？");

```

Comment:
再代入することはないと思うので `const` が良いと思います。

```suggestion
    const name = prompt("日本の首都は？");
```